scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +153,35 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeat amount of times.
+    """
+
+    def __init__(self,
+                 operation: str,
+                 configuration: Dict,
+                 repetitions: int = 0) -> None:
+        """Initialize the repeated iterate if repetitions is not present
+          or set to below 0 it will do an infinite loop."""
```

Comment:
Arguments documentation missing. See for instance: https://github.com/google/scaaml/blob/main/scaaml/capture/input_generators/input_generators.py#L44

Example use missing.

Reply:
But wouldn't they be in the configuration Dict?

Reply:
The documentation is not in the configuration dictionary. Where can a user find documentation of what happens when they set `repetitions` to zero, negative number, positive number...? This is a non-trivial thing that is not documented anywhere. We could add that documentation to the whole `build_attack_points_iterator` function. But it needs to be somewhere.

Reply:
It is in the init doc string